solr/core/src/test/org/apache/solr/search/TestQueryLimits.java

```diff
@@ -76,7 +76,17 @@ public void testQueryLimits() throws Exception {
       rsp =
           solrClient.query(
               COLLECTION,
-              params("q", "id:*", "sort", "id asc", "facet", "true", "facet.field", "val_i"));
+              params(
+                  "q",
+                  "id:*",
+                  "sort",
+                  "id asc",
+                  "facet",
+                  "true",
+                  "facet.field",
+                  "val_i",
+                  "multiThreaded",
+                  "false"));
```

Comment:
#2590 proposes to explicitly not (yet) support multi-threading when query limits are used, and with that then the test change here could be reverted.